packages/sdk/shell/src/testing/shell-manager.ts

```diff
@@ -59,22 +61,22 @@ export class ShellManager extends ScopedShellManager {
 
   async resetDevice() {
     const handleDialog = async (dialog: Dialog) => {
-      if (dialog.type() === 'confirm') {
+      if (dialog.type() === RESET) {
```

Comment:
im not sure those two constants are related: one is dialog type and the other one is what the user is expected to type into the input